### PR TITLE
Update Resources element/Image docs to include supported file formats

### PR DESCRIPTION
**Pull request by jargil**

### CODE REVIEW

File: docs/manifest/resources.md [hunk 1]
```diff
@@ -49,6 +49,14 @@ The following additional sizes are also supported, but not required.
...
 - 64x64
 
+The following image/file extensions are supported:
+- bmp
+- gif
+- exif
+- jpg
+- tiff
+- png
+
```
Comment (zlatko-michailov) [inline docs/manifest/resources.md:59]: In the Taos example, I see a lot of .svg icons. The browser certainly supports it. Hopefully the native clients can support it too.

File: docs/manifest/resources.md [hunk 2]
```diff
@@ -49,6 +49,14 @@ The following additional sizes are also supported, but not required.
 - 48x48
 - 64x64
 
+The following image/file extensions are supported:
+- bmp
+- gif
+- exif
+- jpg
+- tiff
```
Comment (zlatko-michailov) [inline docs/manifest/resources.md:57]: I take your word for all these formats.

**Review by zlatko-michailov — APPROVED** [review]

File: docs/manifest/resources.md [hunk 3]
```diff
@@ -49,6 +49,14 @@ The following additional sizes are also supported, but not required.
...
+- png
```
Comment (AlexJerabek) [inline docs/manifest/resources.md:58]: Changing to confirm to style standards. Also, I alphabetized the list, but were those formats in that order for a reason?

```suggestion
The following image file formats are supported.
- BMP
- EXIF
- GIF
- JPG
- PNG
- TIFF
```

**Comment by AlexJerabek** [comment]
Tagging @Rick-Kirkham for awareness, in case other docs should also be updated.

**Comment by jargil** [comment]
> Thanks for making this change. I have some style suggestions. I can either merge them on your behalf, or if you think something needs to be different, you can make the necessary changes.

Thank you, Alex. It would be helpful if you could merge them on my behalf.

**Review by AlexJerabek — APPROVED** [review]

**Pull request merged**